src/lib/core/DataModelTypes.h

```diff
@@ -58,6 +58,8 @@ constexpr EndpointId kRootEndpointId    = 0;
 constexpr ListIndex kInvalidListIndex   = 0xFFFF; // List index is a uint16 thus 0xFFFF is a invalid list index.
 constexpr KeysetId kInvalidKeysetId     = 0xFFFF;
 
+constexpr uint64_t kInvalidIcId = 0;
```

Comment:
This needs to be clearly documented as being a provisional value that might need to change as the spec for this is sorted out.